app/src/main/java/com/gophillygo/app/data/DestinationRepository.java

```diff
@@ -92,21 +121,71 @@ protected Void doInBackground(Void... voids) {
     }
 
     public LiveData<Resource<List<Destination>>> loadDestinations() {
-        return new AttractionNetworkBoundResource<Destination>(webservice, destinationDao, eventDao) {
+        LiveData<Resource<List<Destination>>> data = new AttractionNetworkBoundResource<Destination>(webservice, destinationDao, eventDao) {
             @NonNull
             @Override
             protected LiveData<List<Destination>> loadFromDb() {
                 return destinationDao.getAll();
             }
         }.getAsLiveData();
+        return addAttractionFlags(data, false);
     }
 
     public LiveData<Resource<List<Event>>> loadEvents() {
-        return new AttractionNetworkBoundResource<Event>(webservice, destinationDao, eventDao) {
+        LiveData<Resource<List<Event>>> data = new AttractionNetworkBoundResource<Event>(webservice, destinationDao, eventDao) {
             @NonNull @Override
             protected LiveData<List<Event>> loadFromDb() {
                 return eventDao.getAll();
             }
         }.getAsLiveData();
+        return addAttractionFlags(data, true);
```

Comment:
If possible, would prefer to not wrap here with `MutableLiveData`, and rather modify the database queries to include the flag as an attraction property, as is done currently with the related destination name being set on events when queried.